src/validators/regex.ts

```diff
@@ -0,0 +1,17 @@
+import * as SCWRegex from '@scaleway/regex'
+import { ValidatorFn } from './types'
+
+const validator: ValidatorFn<string, keyof typeof SCWRegex> = regex => {
```

Comment:
I think we should allow regex as string instead, this would allow custom regex to be used and the usage does not change that much

```
import { alphanumLowercase } from '@scaleway/regex' 

...
<TextBoxField regex={alphanumLowercase} />
<TextBoxField regex={/toto.*/} />
<TextBoxField regex={new RegExp(`toto${var}$`)} />

```

Reply:
I think we need to provide a simple way to use our regex from our lib. Otherwise the user can provide its own validate prop ? IK my way require updates when we update scaleway/regex ? What do you think about this ?

Reply:
Not a fan of passing key as string in props, look at scw-ui with colors, it's going to be a shit show so fast :)